res/res_pjsip/location.c

```diff
@@ -1135,11 +1137,102 @@ static int cli_contact_print_body(void *obj, void *arg, int flags)
 		ast_sip_get_contact_short_status_label(status ? status->status : UNKNOWN),
 		(status && (status->status == AVAILABLE)) ? ((long long) status->rtt) / 1000.0 : NAN);
 
+	if (flags & AST_RETRIEVE_FLAG_ALL) { /*  If we are showing list, no need do detailed */
```

Comment:
Since AST_RETRIEVE_FLAG_ALL is part of the sorcery ast_sorcery_retrieve_flags enum, you should probably adjust the changed prototypes or no one will know what flags are supported.